patterns/footer-portfolio.php

```diff
@@ -58,7 +58,7 @@
 
 <!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
 <div class="wp-block-group"><!-- wp:paragraph {"fontSize":"small"} -->
-<p class="has-small-font-size">© 2024 Twenty Twenty Four</p>
+<p class="has-small-font-size"><?php echo esc_html_x( '© 2024 Twenty Twenty Four', '', 'twentytwentyfour' ); ?></p>
```

Comment:
use like that `© 2024 Twenty Twenty-Four` and give context

Reply:
Twenty Twenty Four is probably something we don't want to translate. This string should be processed so the copyright year comes from a variable and not part of the translatable string.

Reply:
I think this should be the site title block.